src/TesApi.Web/BatchScheduler.cs

```diff
@@ -344,6 +344,25 @@ private async Task DeleteBatchTaskAndOrJobAndOrPoolIfExists(IAzureProxy azurePro
             }
         }
 
+        /// <summary>
+        /// Creates a wget command to robustly download a file
+        /// </summary>
+        /// <param name="url">URL to download</param>
+        /// <param name="localFilename">Filename for the output file</param>
+        /// <param name="setExecutable">Whether the file should be made executable or not</param>
+        /// <returns></returns>
+        private string CreateWgetCommand(string url, string localFilename, bool setExecutable = false)
+        {
+            string command = $"wget --https-only --timeout=20 --waitretry=1 --tries=5 --retry-connrefused --continue -O '{localFilename}' '{url}'";
```

Comment:
Is this enough retries to change the way we download onto the compute nodes?

Reply:
There are 5 retries; worst case scenario is 100 seconds of timeouts.  How many retries were you thinking?

Reply:
The task runner retries 9 times before giving up.